CONTRIBUTING.md

```diff
@@ -220,7 +220,7 @@ fi
 -   For single line statements and associated output, use Python console syntax (pycon):
 
 ```python
->>> pipeline.dispatch(**params) # Dispatching a workflow returns a unique dispatch id.
+>>> ct.dispatch(pipeline)(**params) # Dispatching a workflow returns a unique dispatch id.
```

Comment:
If we go down the `ct.dispatch(pipeline, pipeline_args)` path one major issue is that, say the user wants to pass some positional arguments and some keyword arguments, how can he do that? `ct.dispatch(pipeline, pipeline_args = {1, x: 3, y: 4}` doesn't work as `1` doesn't have a key. Then they'll have to pass positional arguments as different than keyword arguments so they'll do `ct.dispatch(lattice, pos_args=[1, 2], kw_args={x: 3, y: 4}`. Which already looks too convoluted for a simple task like passing in inputs. Any variation of this like `ct.dispatch(lattice, args=(some_pos_args, some_kwargs)` will result in forcing the user to pass their arguments in a certain manner which we want to avoid.

In `ct.dispatch(pipeline)(*args, **kwargs)` they can pass their arguments how they would in a normal function.

Reply:
Ah great answer, thanks!